lib/matplotlib/backends/backend_qt5.py

```diff
@@ -118,6 +118,11 @@ def _create_qApp():
                 if display is None or not re.search(r':\d', display):
                     raise RuntimeError('Invalid DISPLAY variable')
 
+            try:
+                QtWidgets.QApplication.setAttribute(
+                    QtCore.Qt.AA_EnableHighDpiScaling)
+            except AttributeError:
+                pass
```

Comment:
Would add a comment `# QtCore.Qt.AA_EnableHighDpiScaling only available on Qt >= 5.6` to clarify why we expect an AttributeError.

Reply:
sure.